### PR TITLE
Add support for Nepali (ne) language

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/cSuhVSSq/2550-add-nepalese-translation-option-to-whitehall

### CODE REVIEW

File: config/locales/ne.yml [hunk 1]
```diff
@@ -0,0 +1,683 @@
...
+        minister:
+        speaker:
+      delivered_on:
+      delivery_title:
+        minister:
+        speaker:
+      written_on:
+    type:
+      about:
+        few:
```
Comment (brucebolt) [inline config/locales/ne.yml:125]: I _think_ (based on the [Unicode rules](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html#ne)), that Nepali only uses `one` and `other` for it's pluralisation.

**Review by brucebolt — COMMENTED** [review]
This looks good @MuriloDalRi.  I've added one comment as I don't think we need all the plural forms for Nepali.

File: config/locales/ne.yml (hunk 1 above)
Comment (MuriloDalRi) [reply config/locales/ne.yml]: Does it look ok now?

Comment (brucebolt) [reply config/locales/ne.yml]: Perfect 🎉

**Review by brucebolt — APPROVED** [review]

**Pull request merged**